src/MongoDB/WriteConcern.c

```diff
@@ -265,9 +252,7 @@ static PHP_METHOD(WriteConcern, getWtimeout)
 		return;
 	}
 
-	/* Note: PHP currently enforces that wimeoutMS is a 32-bit integer, so
-	 * casting will never truncate the value. This may change with PHPC-1411. */
-	RETURN_LONG((int32_t) mongoc_write_concern_get_wtimeout_int64(intern->write_concern));
+	RETURN_LONG(mongoc_write_concern_get_wtimeout_int64(intern->write_concern));
```

Comment:
I suppose there's a risk of truncation here if a user on a 32-bit platform somehow created a WriteConcern with a 64-bit `wtimeout` value; however, that doesn't seem possible so long as [`_mongoc_uri_build_write_concern()`](https://github.com/mongodb/mongo-c-driver/blob/1.15.1/src/libmongoc/src/mongoc/mongoc-uri.c#L1606) always reads it as a 32-bit integer (no matter the architecture).

Reply:
With `_mongoc_uri_build_write_concern` now always reading a 64-bit integer, the question is how to proceed here: returning a string would fix this, but would technically constitute a BC break.

Reply:
Added comment in [PHPC-1411](https://jira.mongodb.org/browse/PHPC-1411?focusedCommentId=2570617&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-2570617) to note the documentation change outside of this PR.